package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "vivi-type-proxy",
-  "version": "1.0.0",
+  "name": "type-proxy",
+  "version": "1.0.1",
```

Comment:
I find it's usually best to only bump the version directly on master, after merging, but this early on it doesn't matter